build.sbt

```diff
@@ -60,10 +60,11 @@ lazy val `pekko-http-cors` = project
     Compile / packageBin / packageOptions += Package.ManifestAttributes(
       "Automatic-Module-Name" -> "ch.megard.pekko.http.cors"
     ),
-    libraryDependencies += "org.apache.pekko" %% "pekko-http" % pekkoHttpVersion cross CrossVersion.for3Use2_13,
-    libraryDependencies += "org.apache.pekko" %% "pekko-stream" % pekkoVersion % Provided cross CrossVersion.for3Use2_13,
-    libraryDependencies += "org.apache.pekko" %% "pekko-http-testkit" % pekkoHttpVersion % Test cross CrossVersion.for3Use2_13,
-    libraryDependencies += "org.apache.pekko" %% "pekko-stream-testkit" % pekkoVersion % Test cross CrossVersion.for3Use2_13,
+    libraryDependencies += "org.apache.pekko" %% "pekko-http" % pekkoHttpVersion,
+    libraryDependencies += "org.apache.pekko" %% "pekko-stream" % pekkoVersion % Provided,
+    libraryDependencies += "org.apache.pekko" %% "pekko-http-testkit" % pekkoHttpVersion % Test,
+    libraryDependencies += "org.apache.pekko" %% "pekko-stream-testkit" % pekkoVersion % Test,
+    libraryDependencies += "org.scala-lang.modules" %% "scala-java8-compat" % "1.0.2",
```

Comment:
Pekko only provides `scala-java8-compat` for Scala 2.12. Since this project still supports Scala 2.12 and uses classes like `scala.compat.java8.OptionConverters` we can just add the dependency explicitly.